### PR TITLE
add staging option to deploy app to reflect prod

### PR DESCRIPTION
**Pull request by Seeker1911**

### CODE REVIEW

File: rest-api/tools/deploy_app.sh [hunk 1]
```diff
@@ -42,7 +42,7 @@ if [[ $(git status --porcelain) ]]; then
 fi
 
 UPDATE_TRACKER=tools/update_release_tracker.py
-if [ "${PROJECT}" == "all-of-us-rdr-prod" ]
+if [ "${PROJECT}" == "all-of-us-rdr-prod" ] || [ "${PROJECT}" == "all-of-us-rdr-staging" ]
```
Comment (danqrodney) [inline rest-api/tools/deploy_app.sh]: This should change to use config_staging.json... we don't want to use prod config, just app_prod.yaml.

File: rest-api/tools/deploy_app.sh [hunk 2]
```diff
@@ -154,7 +154,7 @@ then
 
   if [ "$TARGET" == "app" ] || [ "$TARGET" == "all" ]
   then
-    if [ "${PROJECT}" = "all-of-us-rdr-prod" ]
+    if [ "${PROJECT}" = "all-of-us-rdr-prod" ] || [ "${PROJECT}" = "all-of-us-rdr-staging" ]
```
Comment (danqrodney) [inline rest-api/tools/deploy_app.sh]: I'm not sure I'd merge this into master, as we don't want staging normally to use app_prod.yaml... only during load tests.

If we do want to merge something into master here, we should probably add a --use_prod_yaml flag that we check here instead.

File: rest-api/tools/deploy_app.sh (hunk 1 above)
Comment (Seeker1911) [reply rest-api/tools/deploy_app.sh]: done

File: rest-api/tools/deploy_app.sh (hunk 2 above)
Comment (Seeker1911) [reply rest-api/tools/deploy_app.sh]: done

File: rest-api/tools/deploy_app.sh [hunk 3]
```diff
@@ -18,6 +18,7 @@ while true; do
     --version) VERSION=$2; shift 2;;
     --deploy_as_version) DEPLOY_AS_VERSION=$2; shift 2;;
     --target) TARGET=$2; shift 2;;
+    --use_prod_yaml) USE_PROD_YAML='true'; shift 2;;
```
[the remark this answers is not in the document]
Comment (Seeker1911) [reply rest-api/tools/deploy_app.sh]: done

**Pull request merged**